logic.js

```diff
@@ -0,0 +1,100 @@
+  const Tweeter = function () {
+    let _posts = [
```

Comment:
Try to use `const` as much as possible. Here you are not changing the fact the _posts is an array, you are just editing the data inside the array. So it can be a `const`